.gitlab-ci.yml

```diff
@@ -29,6 +29,7 @@ unstable:
     - master
     - dev
     - /^task-.*$/
+    - FORMAT
```

Comment:
Remove this file at all since we moved to GitHub

Reply:
Done